docs/token-contracts/fa2/2-fa2-nft-tutorial.md

```diff
@@ -0,0 +1,533 @@
+---
+id: 2-fa2-nft-tutorial
+title: Non-Fungible Tokens on Tezos Using FA2
+sidebar_label: FA2 NFT Tutorial LIGO
+---
+
+This tutorial shows how to originate and interact with the FA2 NFT contract
+implementation. The tutorial uses a pre-compiled FA2 NFT contract written in
+[LIGO](https://ligolang.org/) smart contract language and a command line interface
+(CLI) to originate and interact with the NFT contracts either on the
+[Flextesa](https://tezos.gitlab.io/flextesa/) sandbox or Tezos testnet (Carthagenet).
+
+**Disclaimer:** We highly recommend users read the additional resources above and
+take necessary precautions before following this tutorial and interacting with
+experimental technology. Use this tutorial at your own risk.
+
+## Introduction
+
+### What is FA2 (TZIP-12)?
+
+FA2 refers to a token standard ([TZIP-12](https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-12/tzip-12.md))
+on Tezos. FA2 proposes a unified token contract interface, supporting a wide range
+of token types. The FA2 provides a standard API to transfer tokens, check
+token balances, manage operators (addresses that are permitted to transfer tokens
+on behalf of the token owner) and manage token metadata.
+
+### What is a Non-Fungible Token (NFT)?
+
+NFT (non-fungible token) is a special type of cryptographic token which represents
```

Comment:
"_A_ NFT (non-fungible token) is a ~special~ type _of_ ~cryptographic~ token"

Reply:
yes, and I'd use "**An** NFT"